src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -103,7 +106,7 @@ class ChatSwitcherView extends React.Component {
         if (prevProps.isSidebarAnimating
                 && !this.props.isSidebarAnimating
         ) {
-            this.textInput.focus();
+            // this.textInput.focus();
```

Comment:
Is there a reason we're removing this focus check now? If you did intend to remove this, please remove the code instead of just commenting it out.

Reply:
Because of it, the focus can't stay on `ReportActionCompose`.
Also, that condition doesn't work except when a keyboard event fires.